### ISSUE TITLE
Pre-work Checklist: Developer: Steve Carter

### ISSUE THREAD

**Issue by scarter31191** [hackforla/website#2937]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Action Items
- [x] Add the "role: front end" or "role: back end" label*(s) to this issue and remove the "role missing" label
- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your Gmail address with the team lead so you can be added to the Google Drive
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0) 
- [x] Confirm with the team lead that they have added you to the zoom meeting calendar invites and GitHub repository
- [x] Self Assign this issue (gear in right side panel)
- [x] Add this issue to Projects: Project Board (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
  - [x] Use the information from part 2 to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
- [x] Once you take a good first issue, you will provide estimates and progress reports. 
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment.
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue.  This update should be done every week for every issue.  The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
    - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
    - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Progress through issues with increasing complexity in the following order:
  - Good first issue
  - Good second issue
  - Small 
  - Medium 
  - Large
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points
- Then, add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting
- You may also add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question
- Lastly, you can post your question in the hfla slack channel and link the issue you're working on, so other volunteers can see and respond

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by ExperimentsInHonesty** [comment]
@scarter31191 Hi! I noticed that you self assigned a good first issue 
- #2720, but you have not annotated it as required in instructions above.  See where it says:

- [x] Once you take a good first issue, you will provide estimates and progress reports. 
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment.
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```

**Comment by phuonguvan** [comment]
@scarter31191 Hey! This is the design team lead and I was wondering if you are still working on this project. Bonnie told me to update you guys so let me know your progress or availability as soon as possible!

**Comment by SAUMILDHANKAR** [comment]
@phuonguvan Steve is still working on this issue. Once the pending action items are checked-off, this issue would be closed. Thank you.

**Comment by scarter31191** [comment]
I'm done with this issue. How do I close this issue?

**Comment by SAUMILDHANKAR** [comment]
@scarter31191 Hi Steve, good job in finishing up with all action items. I just wanted to check if you have already shared a weekly progress report in the following format:
Provide Update
1. Progress
2. Blockers
3. Availability
4. ETA

I looked up all your issues but couldn't find one. Please let me know if I have missed. Once that is done you can just click close issue button at the bottom of your screen. Thanks.
![Closing the issue](https://user-images.githubusercontent.com/843538/161402034-7d3800f3-373e-48eb-ab69-aa3505f96e78.PNG)

**Comment by SAUMILDHANKAR** [comment]
@scarter31191 Thanks for providing your weekly progress update in your latest issue. Looks like all your action items are complete. Great job! Please close this issue by clicking on the close issue button at the bottom of this issue. Thank you.

**Comment by SAUMILDHANKAR** [comment]
@scarter31191 Hi Steve, please close this issue by clicking on the close issue button at the bottom of this issue. Will leave a slack message as well. Thank you.

**Comment by HackforLABot** [comment]
## Activity Log: scarter31191
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- scarter31191 opened: https://github.com/hackforla/website/issues/2937 at 03/07/2022, 07:59 PM PST
- scarter31191 assigned: https://github.com/hackforla/website/issues/2937#issuecomment-1061380867 at 03/08/2022, 08:06 PM PST
- scarter31191 assigned: https://github.com/hackforla/website/issues/2720#issuecomment-1059946186 at 03/09/2022, 09:24 PM PST
- scarter31191 commented on: https://github.com/hackforla/website/issues/2720#issuecomment-1064740927 at 03/10/2022, 07:38 PM PST
- scarter31191 opened: https://github.com/hackforla/website/pull/2973 at 03/12/2022, 07:53 PM PST
- scarter31191 commented on: https://github.com/hackforla/website/pull/2970#issuecomment-1066146997 at 03/13/2022, 10:31 AM PDT
- scarter31191 submitted review: https://github.com/hackforla/website/pull/2970#pullrequestreview-908275985 at 03/13/2022, 09:07 PM PDT
- scarter31191 closed as completed: https://github.com/hackforla/website/issues/2272#event-6232468901 at 03/13/2022, 09:20 PM PDT
- scarter31191 reopened: https://github.com/hackforla/website/issues/2272#event-6232468901 at 03/13/2022, 09:20 PM PDT
- scarter31191 assigned: https://github.com/hackforla/website/issues/2271#issuecomment-926945747 at 03/13/2022, 09:26 PM PDT
- scarter31191 commented on: https://github.com/hackforla/website/issues/2271#issuecomment-1066331268 at 03/13/2022, 09:33 PM PDT
- scarter31191 commented on: https://github.com/hackforla/website/issues/2271#issuecomment-1067505091 at 03/14/2022, 07:45 PM PDT
- scarter31191 issue closed by PR 2973: https://github.com/hackforla/website/issues/2720#event-6246521267 at 03/15/2022, 02:17 PM PDT
- scarter31191 PR was merged: https://github.com/hackforla/website/pull/2973#event-6246521304 at 03/15/2022, 02:17 PM PDT
- scarter31191 opened: https://github.com/hackforla/website/pull/2986 at 03/15/2022, 08:53 PM PDT
- scarter31191 commented on: https://github.com/hackforla/website/pull/2986#issuecomment-1069938719 at 03/16/2022, 07:38 PM PDT
- scarter31191 issue closed by PR 2986: https://github.com/hackforla/website/issues/2271#event-6266965149 at 03/18/2022, 08:24 AM PDT
- scarter31191 PR was merged: https://github.com/hackforla/website/pull/2986#event-6266965171 at 03/18/2022, 08:24 AM PDT
- scarter31191 assigned: https://github.com/hackforla/website/issues/2942#issuecomment-1062079991 at 03/24/2022, 06:14 PM PDT
- scarter31191 commented on: https://github.com/hackforla/website/issues/2942#issuecomment-1078559721 at 03/24/2022, 06:15 PM PDT
- scarter31191 opened: https://github.com/hackforla/website/pull/3013 at 03/25/2022, 07:01 PM PDT
- scarter31191 issue closed by PR 3013: https://github.com/hackforla/website/issues/2942#event-6311976071 at 03/26/2022, 08:12 AM PDT
- scarter31191 PR was merged: https://github.com/hackforla/website/pull/3013#event-6311976079 at 03/26/2022, 08:12 AM PDT
- scarter31191 commented on: https://github.com/hackforla/website/pull/3013#issuecomment-1079719954 at 03/26/2022, 08:48 AM PDT
- scarter31191 assigned: https://github.com/hackforla/website/issues/2902#issuecomment-1053998283 at 03/26/2022, 09:22 AM PDT
- scarter31191 assigned: https://github.com/hackforla/website/issues/2902#issuecomment-1053998283 at 03/26/2022, 09:22 AM PDT
- scarter31191 opened: https://github.com/hackforla/website/pull/3015 at 03/26/2022, 09:54 AM PDT
- scarter31191 commented on: https://github.com/hackforla/website/issues/2937#issuecomment-1079777951 at 03/26/2022, 02:28 PM PDT
- scarter31191 commented on: https://github.com/hackforla/website/pull/3012#issuecomment-1079817233 at 03/26/2022, 07:14 PM PDT
- scarter31191 submitted review: https://github.com/hackforla/website/pull/3012#pullrequestreview-922389396 at 03/26/2022, 07:17 PM PDT
- scarter31191 commented on: https://github.com/hackforla/website/pull/3015#issuecomment-1080045464 at 03/27/2022, 04:40 PM PDT
- scarter31191 commented on: https://github.com/hackforla/website/pull/3015#issuecomment-1080048730 at 03/27/2022, 05:00 PM PDT
- scarter31191 commented on: https://github.com/hackforla/website/pull/3017#issuecomment-1080089094 at 03/27/2022, 06:30 PM PDT
- scarter31191 submitted review: https://github.com/hackforla/website/pull/3017#pullrequestreview-922558097 at 03/27/2022, 07:29 PM PDT
- scarter31191 issue closed by PR 3015: https://github.com/hackforla/website/issues/2902#event-6322778658 at 03/28/2022, 06:42 PM PDT
- scarter31191 issue closed by PR 3015: https://github.com/hackforla/website/issues/2902#event-6322778658 at 03/28/2022, 06:42 PM PDT
- scarter31191 PR was merged: https://github.com/hackforla/website/pull/3015#event-6322778666 at 03/28/2022, 06:42 PM PDT
- scarter31191 commented on: https://github.com/hackforla/website/pull/3020#issuecomment-1082560210 at 03/29/2022, 07:40 PM PDT
- scarter31191 assigned: https://github.com/hackforla/website/issues/2625#issuecomment-1001218168 at 03/31/2022, 07:02 PM PDT
- scarter31191 commented on: https://github.com/hackforla/website/issues/2625#issuecomment-1085328985 at 03/31/2022, 07:05 PM PDT
- scarter31191 commented on: https://github.com/hackforla/website/issues/2625#issuecomment-1093650832 at 04/08/2022, 08:47 PM PDT
- scarter31191 commented on: https://github.com/hackforla/website/issues/2625#issuecomment-1098655834 at 04/13/2022, 07:46 PM PDT
- scarter31191 closed as completed: https://github.com/hackforla/website/issues/2937#event-6444851170 at 04/17/2022, 10:54 AM PDT
- scarter31191 commented on: https://github.com/hackforla/website/issues/2625#issuecomment-1154661785 at 06/13/2022, 08:13 PM PDT
- scarter31191 unassigned: https://github.com/hackforla/website/issues/2625#issuecomment-1166596854 at 01/08/2023, 10:34 AM PST
